frontend/src/profiles/ProfileDetails.js

```diff
@@ -25,10 +26,12 @@ export default function ProfileDetails(){
             <MenuAppBar pagename="Profile details"/>
             <ProfileDetailsStyled>
                 <div>
+                    <Card className={classes.card}>
                     <h1>{profile.name}</h1>
                     <h3>{profile.location}</h3>
                     <h4>{profile.birthday}</h4>
                     <SkillList skills={profile.skills}/>
```

Comment:
-semantic gap_ between `h1` and `h2`. Maybe use `address` and `time` tag instead?